### ISSUE TITLE
Breaking : Latest Chrome breaks creating Browser object, steps to reproduce included. 

### ISSUE THREAD

**Issue by dogeatdog** [atroche/rust-headless-chrome#367]
Hi,

After updating Chrome on my Linux machine to 110.0.5481.77 (Official Build) (64-bit) the Browser object doesn't seem to be working anymore. 

![2023-02-10_10-00](https://user-images.githubusercontent.com/11708786/218048622-69817660-57fb-47c7-ae41-90eda6c460ab.png)

Neither new with LaunchOptions or default seems to function, both return an error (due to Compat { error: Timeout }). Since literally nothing changed overnight, apart from Chrome being updated, I suspect an update is needed.

I've looked at the implementation of the  default_executable method and it doesn't seem to contain the current executable for chrome anymore : 

"google-chrome-stable",
        "chromium",
        "chromium-browser",
        "chrome",
        "chrome-browser",

I've tested my theory by ln google-chrome-stable to chrome, but unfortunately that doesn't remedy the issue :( 

Are you seeing the same issue or do you need more info to reproduce this error ?

**Comment by dogeatdog** [comment]
As an extra : 

`  println!("Initializing browser");
    let browser = Browser::default()?;
    println!("Get initial tab");
    let tab = browser.wait_for_initial_tab()?;`

Results in : 

`Initializing browser
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Timeout', /home/dennis/.cargo/registry/src/github.com-1ecc6299db9ec823/headless_chrome-0.9.0/src/browser/mod.rs:101:38`

**Comment by dogeatdog** [comment]
Just to make sure it wasn't caused by my fork of the main project I quickly created a new Rust project using the main crate (atroche one, 1.0.4).

Using the sample code (the wikipedia screenshot thing) I was able to reproduce the exact same issue : 

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: The event waited for never came', /home/dennis/.cargo/git/checkouts/rust-headless-chrome-fd32a97e7a7fa8b4/11b29fc/src/browser/mod.rs:111:38

The code where it breaks is when the Browser object gets initialized. 

![2023-02-10_09-07](https://user-images.githubusercontent.com/11708786/218036792-f73b8e6a-d1e5-43a1-8978-ba258a671cdc.png)

**Comment by dogeatdog** [comment]
For what it's worth I found a workaround for now. 

I downloaded the previous version of Chrome for Linux from https://chromium.cypress.io/linux/stable/109.0.5414.119 and extracted that to a folder.

Then I adjusted the LaunchOptions to use that binary : 

![image](https://user-images.githubusercontent.com/11708786/218096895-ae996d93-9cab-424d-ba89-d63e22390e74.png)

This works. And proves it's really Chrome 1.10.x breaking everything.

**Comment by dogeatdog** [comment]
For the sake of reproducibility I did a couple of tests today on a Linode Ubuntu 22.04 box, to make sure it wasn't my environment nor my network.

- I downloaded Chrome version 109, 110, 111 and 112 from the chromium.cypress.io site
- Then used the code from the readme file (the wikipedia test)
-  And updated the path in the LaunchOptionsBuilder for each of the 4 tests.

![image](https://user-images.githubusercontent.com/11708786/218322683-62d8e486-5dd4-422f-aaec-abb1f3ef0ab7.png)


Results : 

![image](https://user-images.githubusercontent.com/11708786/218322101-f525cae0-2907-4fe9-8df3-7619b0e5f414.png)

![image](https://user-images.githubusercontent.com/11708786/218322229-4d8fd9ab-6446-480e-a5b5-ba911298efbd.png)

I even tried with the two next versions : 

![image](https://user-images.githubusercontent.com/11708786/218322324-8b7aab5f-1f50-4875-92a4-622c16f422d9.png)

![image](https://user-images.githubusercontent.com/11708786/218322579-c8eeeef3-8b0b-43e1-ad16-877858c0ecab.png)

In short, it doesn't look good. Everything from Chrome 110.x and higher breaks the current implementation of rust-headless-chrome.

**Comment by Billy-Sheppard** [comment]
Can confirm this is an issue, I will try and investigate and isolate the issue.

**Comment by Billy-Sheppard** [comment]
Ok doesn't seem to be too bad, looks like Chrome no longer launches with a tab. Will set up a PR for a fix.

**Comment by cyqsimon** [comment]
Thanks for tackling this issue so quickly. I updated my Chrome browser today and ran straight into this with my personal project.

I tested your patched version and it seems like it's fixed.

**Comment by dogeatdog** [comment]
@Billy-Sheppard Works fine. Took a bit longer for me because I had to integrate your fix into my fork, but works like a charm. Thanks for fixing this !

**Comment by dogeatdog** [comment]
Lol. I just realized just using let tab = browser.new_tab()?; would have fixed all my problems from the get-go :)

Welp.
